ooyala_player/ooyala_player.py

```diff
@@ -189,6 +189,8 @@ def student_view(self, context):
 
         JS_URLS = [
             self.local_resource_url(self, 'public/build/player_all.min.js'),
+            self.local_resource_url(self, 'public/js/vendor/ooyala/bit_wrapper.js'),
+            self.local_resource_url(self, 'public/js/vendor/ooyala/bitmovinplayer.swf'),
```

Comment:
This is not a JavaScript resource, please embed it directly in the HTML.

Reply:
@xitij2000 I verified it on Firefox, Chrome and IE and it is working fine on all browsers.
Do you want me to add it in ooyala_player.html as 
<Link href='public/js/vendor/ooyala/bitmovinplayer.swf' />

Reply:
@zamanafzal Nope, you can embed it as:
```html
<object width="0" height="0">
    <param name="bitmovin" value="{path_to_bitmovin}">
    <embed src="{path_to_bitmovin}" width="0" height="0">
    </embed>
</object>
```

Reply:
Re: the working/not-working on Firefox etc. Since swf files are not supposed to be included as JavaScript in any case, even if it's working now it could break in the future. I am testing on Firefox 63 and 65 and it didn't work in either.